src/components/welcome-screen/WelcomeScreen.Hints.tsx

```diff
@@ -10,7 +10,7 @@ const MenuHint = ({ children }: { children?: React.ReactNode }) => {
   const { WelcomeScreenMenuHintTunnel } = useTunnels();
   return (
     <WelcomeScreenMenuHintTunnel.In>
-      <div className="virgil welcome-screen-decor welcome-screen-decor-hint welcome-screen-decor-hint--menu">
+      <div className="welcome-screen-decor welcome-screen-decor-hint welcome-screen-decor-hint--menu">
```

Comment:
Change hints font to helvetica also
![image](https://github.com/user-attachments/assets/ba915bc6-ae6b-4498-8834-4fd1be8cae54)